### PR TITLE
Correct incorrect cache help description

### PR DESCRIPTION
**Pull request by tomballgithub**
## Description

The help text for the new cache_type configuration has a cut/paste error which errantly mentions the google API.  This corrects that as well as making it clear that there are two choices, mem or file.

## Type of Change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context

Help text should be correct

## Wiki Update

- [ ] This change requires an update to the Wiki.
- [X] This change does not require an update to the Wiki.

## Checklist
- [X] This change follows the code style of this project.
- [X] This change only changes what is necessary to the feature.

### CODE REVIEW

File: start_pokealarm.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def parse_settings(root_path):
                         help='Specify either metric or imperial units to use for distance measurements. ')
     parser.add_argument('-ct', '--cache_type', type=parse_unicode, action='append', default=['mem'],
                         choices=cache_options,
-                        help='Specify a Google API Key to use.')
+                        help='Select cache storage location (mem or file)')
```
Comment (kvangent) [inline start_pokealarm.py]: Change this to `Specify the type of cache to use. Options: ['mem', 'file'] (Default: 'mem')`

File: config/config.ini.example [hunk 2]
```diff
@@ -18,7 +18,7 @@
 #geofence:										# File containing geofence(s) used to filter (default: None)
 #location:										# Location for the manager. 'Name' or 'lat lng' (default: None)
 #locale:										# Language to be used to translate names (default: en)
-#cache_type:                                    # Type of cache to use. (default: 'mem')
+#cache_type:                                    # Type of cache to use: mem or file. (default: 'mem')
```
Comment (kvangent) [inline config/config.ini.example]: Change this to `Specify the type of cache to use. Options: ['mem', 'file'] (Default: 'mem')`

**Comment by tomballgithub** [comment]
I'll change it.  My feedback is that 'mem' and 'file' are not really documented anywhere.  That is why I tried to say this really boils down to the 'cache storage location'.

**Comment by tomballgithub** [comment]
Changes implemented.

**Comment by kvangent** [comment]
The reason I want to avoid saying 'storage location' is because in the future there could be a cache that operates different for whatever reason but still uses a file or mem.

As for documentation, it's mentioned both in [server settings](https://github.com/PokeAlarm/PokeAlarm/wiki/server-settings) and in depth in [Object Caching](https://github.com/PokeAlarm/PokeAlarm/wiki/Object-Caching)

**Comment by tomballgithub** [comment]
It really needs to documented on the github site because the only way I could find the info was go to the discord #announcements and click on a link there to get to the 3.4 notes.  A normal person looking for the info at the PokeAlarm github will be challenged.

**Comment by kvangent** [comment]
The wiki in integrated into the Github site. You can access it from the wiki tab at the top of the page.

**Comment by tomballgithub** [comment]
Thanks, I found: https://github.com/PokeAlarm/PokeAlarm/wiki/Object-Caching

**Pull request merged**